GraphicsToolsUnityProject/ProjectSettings/GraphicsSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!30 &1
 GraphicsSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 14
+  serializedVersion: 13
```

Comment:
I bet this entire file could be reverted since you likely didn't mean to change the render pipeline we point to be default?

Reply:
![image](https://user-images.githubusercontent.com/13305729/199793787-36d094ba-b279-4bc4-8970-0386c51dc723.png)